storage/persistent-storage/persistent-storage-nfs.adoc

```diff
@@ -0,0 +1,33 @@
+[id="persistent-storage-using-nfs"]
+= Persistent Storage Using NFS
```

Comment:
Persistent Storage -> Persistence storage

FYI the other sections for the other types aren't consistent in this sentence case for the main heading.

Reply:
Resolved.

The other sections will be addressed in a separate PR.